bukkit/src/main/java/dev/aurelium/auraskills/bukkit/listeners/PlayerDeath.java

```diff
@@ -26,7 +26,12 @@ public void onPlayerDeath(PlayerDeathEvent event) {
                 if (plugin.configBoolean(Option.ON_DEATH_RESET_SKILLS)) {
                     user.resetSkill(skill);
                 } else if (plugin.configBoolean(Option.ON_DEATH_RESET_XP)) {
-                    user.setSkillXp(skill, 0);
+                    final var ratio = plugin.configDouble(Option.ON_DEATH_RESET_XP_RATIO);
```

Comment:
Remove the final keyword and use double instead of var (var should only be used in the place of long class names).